specification/level-1-version-5/sources/examples.tex

```diff
@@ -0,0 +1,355 @@
+{lst:repressilator}
+{examples/repressilator/repressilator.xml}
+
+\pagebreak
+% ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+% DATA EXAMPLES
+% ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\section{Simulation experiments with dataDescriptions}
+The \hyperref[class:dataDescription]{DataDescription} provides means to use external datasets in simulation experiments. In this section simulation experiments using the \hyperref[class:dataDescription]{dataDescription} are presented.
+
+\subsection{Plotting data with simulations (\texttt{L1V3\_plotting-data-numl.omex})}
+This example demonstrates the use of the \hyperref[class:dataDescription]{DataDescription} and \hyperref[class:dataSource]{DataSource} to load external data in SED-ML. In the example a \hyperref[class:model]{model} is simulated (using a \hyperref[class:uniformTimeCourse]{uniformTimeCourse} simulation) and the simulation results are plotted. In addition data is plotted using the \hyperref[class:dataDescription]{dataDescription} and \hyperref[class:dataSource]{DataSource}), extracting the \token{S1} and \token{time} column from it and renders it. The listed example uses data encoded in \hyperref[sec:dataFormatNUML]{NuML} as \hyperref[sec:dataFormatURN]{format} (\code{urn:sedml:format:numl}). This document can be found at \url{https://sed-ml.org/examples/L1V3/L1V3_plotting-data-numl/plotting-data-numl.xml}, and an OMEX version at \url{https://sed-ml.org/examples/L1V3/L1V3_plotting-data-numl.omex}.
+
+The corresponding example using \hyperref[sec:dataFormatCSV]{CSV} (\code{urn:sedml:format:csv}) as format to encode the data is available as \texttt{L1V3\_plotting-data-csv.omex}.
+
+\begin{figure}[ht]
+    \centering
+    \begin{minipage}{0.47\textwidth}
+        \centering
+        \includegraphics[width=1.0\textwidth]{examples/plotting-data-numl/results/sedml_webtools/plot1}
+        \caption{The simulation result from the simulation description given in \lst{plotting-data-numl}. Simulation with SED-ML web tools \citep{bergmann2017sed}.}
+    \end{minipage}\hfill
+    \begin{minipage}{0.47\textwidth}
+        \centering
+        \includegraphics[width=1.0\textwidth]{examples/plotting-data-numl/results/tellurium/plot1}
+        \caption{Simulation with tellurium \citep{tellurium}.}
+    \end{minipage}
+    \label{fig:plotting-data}
+\end{figure}
+
+\myXmlImport{SED-ML document using \SedDataSource and \SedDataDescription}
+{lst:plotting-data-numl}
+{examples/plotting-data-numl/plotting-data-numl.xml}
+
+\pagebreak
+% ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+% REPEATED TASKS
+% ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\section{Simulation experiments with repeatedTasks}
+The \RepeatedTask makes it possible to encode a large number of different simulation experiments. In this section several such simulation experiments are presented.
+
+% ~~~ TIME COURSE PARAMETER SCAN ~~~
+\subsection{Time course parameter scan (\texttt{L1V3\_repeated-scan-oscli.omex})}
```

Comment:
@luciansmith  the example still has L1V3_ in the filename, though it is l1v5 (also holds for the following examples)